nix/m1-support/kernel/default.nix

```diff
@@ -10,7 +10,7 @@
 
     # set a default frequency governor the same way nixos-generate-config does
     # so the necessary bits get properly cross-compiled
-    powerManagement.cpuFreqGovernor = lib.mkDefault "ondemand";
+    powerManagement.cpuFreqGovernor = lib.mkDefault "schedutil";
```

Comment:
Please change the comment to indicate why schedutil was chosen over the default ondemand.

Please also remove the `lib.mkDefault`. If it's present, a user who follows the guide to install a new system will get an error that there is a conflicting definition for the option as `nixos-generate-config` puts ondemand by default.